### DISCUSSION TITLE
Elk renderer not loading properly when using ESM

### DISCUSSION THREAD

**Discussion by RobertWeinmeister** [mermaid-js/mermaid#4593] [Q&A] [answered]
I have a problem when using the elk renderer. It works fine when I include mermaid like this:
```js
<script src="https://cdn.jsdelivr.net/npm/mermaid@10.2.4/dist/mermaid.min.js"></script>
```
But it stops working if I switch to ESM:
```js
<script type="module">
import mermaid from 'https://cdn.jsdelivr.net/npm/mermaid@10.2.4/+esm'
</script>
```
Somewhere along the line it tries to import ```https://cdn.jsdelivr.net/npm/elkjs@0.8.2/lib/elk.bundled.js/+esm```, which fails with a 500. I tried accessing the file from different computers and networks directly, which also fails.

Now, my question is, is this a problem on my side? Or with Mermaid, or the content deliverer?

**Comment by sidharthv96** [comment, marked as answer]
That is a CDN specific issue. 
Please use the official supported URL instead. 

```html
    <script type="module">
      import mermaid from 'https://cdn.jsdelivr.net/npm/mermaid@10/dist/mermaid.esm.min.mjs';
    </script>
```

https://mermaid.js.org/config/usage.html#simple-full-example

**Reply by RobertWeinmeister** [reply]
Thanks, that works.
